app/styles/components/footer.scss

```diff
@@ -112,3 +113,42 @@
   }
 
 }
+
+@media screen and (max-width: $screen-xs) {
```

Comment:
What do the changes to `footer.scss` achieve?

Reply:
I didn't like how the footer took up so much space on medium viewports (374px to 1000px), now that there was a watermark image in the background. So i made it that the logo and copyright copy were inline until xs screens (374px).
![image](https://user-images.githubusercontent.com/75539955/143937000-ee45f124-2f18-44ea-b09e-10e8695f869e.png)